### PR TITLE
fix(jarvis): run docs script branch

### PR DESCRIPTION
**Pull request by p-fernandez**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Allows to run the jarvis script that runs the docs by fixing the running flow and adding a missing dependency.

- **Why this change was needed?** (You can also link to an open issue here)
To allow all the script options to be fully runnable.

- **Other information**:
N/A

### CODE REVIEW

File: scripts/jarvis.js [hunk 1]
```diff
@@ -20,7 +21,8 @@ async function reInstallProject() {
     if (answers.reinstall === 'No') {
       return;
     }
-    await setupProject();
+
+    return await setupProject();
```
Comment (p-fernandez) [inline scripts/jarvis.js:25]: Fixes a lint warning.

File: scripts/jarvis.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 const fs = require('fs');
+const ora = require('ora');
```
Comment (scopsy) [inline scripts/jarvis.js:2]: Could we move the `ora` require to the scope of the function that executes it? The reason we don't have any imports on the root of the file, is that the file will still be executable even if `node_modules` was not installed yet. Mainly so we can have a single entry point for new contributors (jarvis also does pnpm install if node_modules are not installed).

It's a bit confusing nonetheless, so maybe we could even move the 2 piece of code to different files so we have a node-native only file and the other file that could import things from npm. wdyt?

**Review by scopsy — COMMENTED** [review]
Nice catch! I've only added a small note regarding the import on the top of the file 🚀

File: scripts/jarvis.js (hunk 2 above)
Comment (p-fernandez) [reply scripts/jarvis.js:2]: You are right. I totally missed that pattern when doing the fix. Will change.

**Review by scopsy — APPROVED** [review]
Ship it!!! :)

**Pull request merged**